pkg/provider/provider.go

```diff
@@ -117,6 +117,8 @@ type TestEnvironment struct { // rename this with testTarget
 	DaemonsetFailedToSpawn bool
 	ScaleCrUnderTest       []ScaleObject
 	StorageClassList       []storagev1.StorageClass
+	CreatedBy              string
+	PartnerName            string
 }
 
 type MachineConfig struct {
```

Comment:
What is machine here?

Reply:
Its not my addition

Reply:
Or am I misunderstanding you question

Reply:
Ah, my bad. I thought we are in collector repo. All good.